src/index.jsx

```diff
@@ -134,6 +137,7 @@ class IntlCurrencyInput extends Component {
   allowedProps() {
     const allowedProps = { ...this.props }
 
+		delete allowedProps.defaultValue
```

Comment:
I didn't mean to add this to the pull request, but I did anyway. This avoids a warning when using create-react-app. You shouldn't pass defaultValue to a controlled input